### DISCUSSION TITLE
Right Click on Taskbar Icons no longer brings up menu

### DISCUSSION THREAD

**Discussion by johnkarlhill** [valinet/ExplorerPatcher#3509] [Q&A] [answered]
For example, I have Command Prompt pinned to my taskbar.  If I left click on it, it launches.  Prior to the latest release, if I right click, it brought up the menu to Run as Administrator, or launch another instance, and other options.

Windows version 22H2 (22621.3958)
ExplorerPatcher version 22621.3527.65.5

**Comment by milestprower92** [comment]
> For example, I have Command Prompt pinned to my taskbar. If I left click on it, it launches. Prior to the latest release, if I right click, it brought up the menu to Run as Administrator, or launch another instance, and other options.
> 
> Windows version 22H2 (22621.3958) ExplorerPatcher version 22621.3527.65.5

are you using 7+ taskbar tweaker or other similar software?

**Reply by johnkarlhill** [reply]
I am not using any other taskbar programs.

And to clarify....
On a machine that does not have ExplorerPatcher, if I right click on an icon on my taskbar (pinned program not running OR running program), I get the following:
![image](https://github.com/user-attachments/assets/b1538312-f711-4614-ba2e-78c6339a0c78)

On a machine with ExplorerPatcher installed, nothing happens when I right-click an icon on the taskbar:
![image](https://github.com/user-attachments/assets/8ac95d59-180e-47a6-8de4-7d0bc18b7d03)

Previous versions of ExplorerPatcher, I was able to right click on icons and get a menu.
For example, if I have cmd.exe pinned, I used to get this:
![image](https://github.com/user-attachments/assets/952f3a6c-1421-49f0-91f9-692a1d4bfe0a)

**Reply by pyrates999** [reply]
did this happen after you installed this preview update? https://support.microsoft.com/en-us/topic/july-25-2024-kb5040527-os-builds-22621-3958-and-22631-3958-preview-de3e1e24-0c07-4210-9777-8e03a1446bae

Please try installing the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.3_c843465

**Reply by Sungho-Dot** [reply]
It still doesn't work in the last pre-release.

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
`C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI`

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.

If it doesn't show up now, or doesn't work, click on restart file explorer in the bottom left in the EP properties application.

You can also try to open up task manager and kill explorer.exe and then run task manager again and run explorer.exe.

Also try restarting your computer.

**Reply by milestprower92** [reply]
> to run the explorer patcher properties application, run this from the run menu: `C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI`
> 
> Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.
> 
> If it doesn't show up now, or doesn't work, click on restart file explorer in the bottom left in the EP properties application.
> 
> You can also try to open up task manager and kill explorer.exe and then run task manager again and run explorer.exe.
> 
> Also try restarting your computer.

that is not related with this issue.
the issue is with the taskbar icons jumplist menu, not the taskbar context menu

**Reply by pyrates999** [reply]
Please show a screen shot of what you mean then.

**Reply by milestprower92** [reply]
> Please show a screen shot of what you mean then.

![image](https://github.com/user-attachments/assets/c8e55c5d-d7b4-43ca-9570-c8d4b6edf3e7)

**Reply by milestprower92** [reply]
i think that it is caused by 7+ taskbar tweaker or other similar software

**Reply by pyrates999** [reply]
Try to uninstall 7+ taskbar tweaker and see if it is still happening.

**Reply by backforeground** [reply]
Maybe you had the settings in Windows Registry from an older version of ExplorerPatcher that didn't have the option to have the Properties entry in the Taskbar and when you installed the new version, the new version got stuck with the old settings.

What worked for me was to uninstall ExplorerPatcher, delete the ExplorerPatcher settings from registry:
`HKEY_CURRENT_USER\SOFTWARE\ExplorerPatcher`
and reinstall the latest version of ExplorerPatcher and check that the following setting is **disabled**:

`Advanced > Do not change the taskbar context menu [...] `

**Comment by Sungho-Dot** [comment]
The taskbar right-click menu doesn't work.

**Reply by pyrates999** [reply]
So you right click on an already opened application that has been pinned to the taskbar, and nothing shows up?

**Reply by Sungho-Dot** [reply]
Yes!

**Reply by Sungho-Dot** [reply]
It doesn't work when I set the taskbar style to Windows 10. The position is at the top.

**Reply by pyrates999** [reply]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Reply by Sungho-Dot** [reply]
![1](https://github.com/user-attachments/assets/19314055-d610-4117-a60f-04e884e1cba0)
![2](https://github.com/user-attachments/assets/4e59ba08-e5cd-4a8a-a7ea-46be24746bb2)
![3](https://github.com/user-attachments/assets/38353dc7-80f5-4497-825a-d54e803ec026)
![4](https://github.com/user-attachments/assets/0e302eec-48b2-4d09-9b2a-b1fd04b3ca58)
![5](https://github.com/user-attachments/assets/22712cfb-2280-476c-aaa4-f892999ff95f)
![6](https://github.com/user-attachments/assets/9020e7bf-a5b9-4084-8945-f875c7f65acd)
![7](https://github.com/user-attachments/assets/0edeb570-fb40-4468-b8bc-767585691cf4)
![8](https://github.com/user-attachments/assets/f21f9cda-8737-4383-846b-1e8f3e4e6d5a)
![9](https://github.com/user-attachments/assets/3e6db222-e6cf-45db-9d55-f57576acbbdb)
![10](https://github.com/user-attachments/assets/0b2d23fb-5be7-44c8-b291-c50fcf67f06c)
![x](https://github.com/user-attachments/assets/1b1345eb-25b7-4270-9451-480164926bcf)

I have not installed any other software that modifies the UI of Windows.

**Reply by pyrates999** [reply, marked as answer]
If you have the windows 10 taskbar set to a position that is not at the bottom, this is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3515

**Reply by johnkarlhill** [reply]
I can confirm that if I drag the taskbar to the bottom, the "flyout" menus do work.

**Comment by Amrsatrio** [comment]
This works on me. 22621.3958. Right clicking pinned command prompt with no open windows brings up the jump list with two entries: command prompt (to launch a new instance) and unpin from taskbar.

So this is not an EP issue.

**Comment by Gerbil4901** [comment]


**Reply by pyrates999** [reply]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.

**Reply by pyrates999** [reply]
different antivirus call it different names.

**Reply by pyrates999** [reply]
Still a false positive

**Comment by Amrsatrio** [comment]
https://github.com/valinet/ExplorerPatcher/issues/3514
